lxd/storage_buckets.go

```diff
@@ -358,6 +385,17 @@ func storagePoolBucketGet(d *Daemon, r *http.Request) response.Response {
 		bucket.S3URL = u.String()
 	}
 
+	if util.IsRecursionRequest(r) {
+		if len(withEntitlements) > 0 {
+			err = reportEntitlements(r.Context(), s.Authorizer, s.IdentityCache, entity.TypeStorageBucket, withEntitlements, map[*api.URL]auth.EntitlementReporter{entity.StorageBucketURL(effectiveProjectName, bucket.Location, details.pool.Name(), bucket.Name): &bucket.StorageBucket})
+			if err != nil {
+				return response.SmartError(err)
+			}
+		}
+
+		return response.SyncResponse(true, &bucket.StorageBucket)
```

Comment:
this doesnt look like it should be there as we have a (correct) SyncReponseETag call below, and here we have no ETag generated which would be a regression.